scripts/fund-round-creator.sh

```diff
@@ -111,7 +111,7 @@ print_message "fund_store file: $FUND_STORE_FILE"
 print_prompt "Did you ruff formatted the form_runner? (1/0): "
 read -p "" confirm
 
-if [[ "$confirm" == "1" ]]; then
+if [[ "$confirm" == "0" ]]; then
     print_error "Exiting script. Come back after you ruff format the form_runner."
     exit 1
 fi
```

Comment:
Why not just ruff format it for them? 👀  `uv tool run ruff format <dir>` should do the trick (sad that this is needed 😭)

Reply:
:thinking: I thought a bit of ways to automate the ruff-ing. Will give it a throught..

Reply:
even better idea, don't dynamically write python files when we're just exporting data 😬